src/Microsoft.VisualStudio.AppDesigner/Microsoft.VisualStudio.AppDesigner.vbproj

```diff
@@ -14,6 +14,8 @@
     <Description>This package implements the AppDesigner, which is the designer host used for project property pages among other things in Visual Studio</Description>
     <Summary>AppDesigner designer host implementation for Visual Studio project property pages</Summary>
     <PackageTags>Roslyn Project AppDesigner VisualStudio</PackageTags>
+    <!-- By default the SDK sets RootNamespace to the name of the project. Clear it here to avoid prefixing our namespaces. -->
+    <RootNamespace />
```

Comment:
TODO: Consider putting this in the Languages.props.

Reply:
Attempted this but it is set too early. The SDK ends up overriding it.
![image](https://user-images.githubusercontent.com/17788297/178364974-740e7121-cf3e-40d4-8e26-842e6bde6eba.png)